ONSdigital/ras-frontstage
# Updating SEFT wording for respondents

# What and why?
Adding a couple of lines to help inform the user about which files to download and upload when responding to a SEFT survey.
# How to test?
- Deploy this pr to your environment
- Run the acceptance tests tagged as SEFT
- Ensure that the new lines of text appear when in the SEFT survey
# Trello
https://trello.com/c/SidH5mOt/2341-update-seft-download-and-upload-pages-wording
